vic/vic_run/include/vic_def.h

```diff
@@ -60,7 +60,7 @@
 #define MIN_VEGCOVER    0.0001 /**< Minimum allowable vegcover fraction */
 
 /***** Define minimum and maximum values for model timesteps *****/
-#define MIN_SUBDAILY_STEPS_PER_DAY  4
+#define MIN_SUBDAILY_STEPS_PER_DAY  1
```

Comment:
Ah it was a mistake from me to commit this line. However we need to talk about it because I still cannot run on a daily basis (subdaily timestep > 1) without wrong output.
Skype this evening (morning for you :))?